### PR TITLE
Add support for CSS and LESS files by default #8

### PR DESCRIPTION
**Pull request by VovanR**

### CODE REVIEW

**Comment by JaKXz** [comment]
@VovanR awesome contribution 💯

sorry for the delayed response though. Would you mind rebasing your PR? The only conflicts will be Line 65 in `index.js` and the location of the test files -- there's a helper function now that get's the path from the `./test` directory in `./test/helpers/setup.js`. Let me know if you have issues, or I can take over this PR when I have a few minutes to spare!

**Comment by VovanR** [comment]
@JaKXz done.

**Comment by JaKXz** [comment]
@vieron any final thoughts? This should make the 0.3.1 release. 

Thanks again @VovanR!

**Comment by vieron** [comment]
@JaKXz looks good to me! Thanks @VovanR!

@JaKXz I can do both releases (0.3.0 and 0.3.1) in a few hours

**Comment by vieron** [comment]
Now that I was going to merge, I thought this could be a breaking change for some consumers.

Consumers could use specific syntax in `.scss` files following rules defined in their `.stylelintrc`, but not in `.css/.less` files. I can imagine things like normalize.css or other vendor css libraries breaking build pipelines because of stylelint started to lint `.css` files.

What do you think @JaKXz @VovanR ?

**Comment by JaKXz** [comment]
Perhaps there should be another test that makes sure those errors aren't thrown if you have mixed style files?

Though I don't think vendor libraries will actually be linted if they're `import`ed or `require`d so I can't think of a real scenario where this would actually be an issue. Please correct me if I'm wrong!

**Comment by JaKXz** [comment]
If necessary, we can make this a breaking change as well, for 0.4.0

**Comment by VovanR** [comment]
I place vendor libs to `disable` section
```css
/* stylelint-disable */
@import "../bower_components/awesome-bootstrap-checkbox/awesome-bootstrap-checkbox.css";
@import "../bower_components/mdi/css/materialdesignicons.css";
@import "../bower_components/owl.carousel/dist/assets/owl.carousel.css";
@import "../bower_components/owl.carousel/dist/assets/owl.theme.default.css";
@import "../bower_components/fancybox/source/jquery.fancybox.css";
@import "../bower_components/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.css";
/* stylelint-enable */
```
I do not think that this is a breaking change

**Comment by JaKXz** [comment]
@VovanR could you update your fixtures to handle some of the edge cases @vieron mentioned? I  have a feeling you're right and it won't be a breaking change, but it would be nice to make sure.

File: test/index.js [hunk 1]
```diff
@@ -120,6 +120,20 @@ describe('sasslint-loader', function () {
...
+  it('should work with CSS, SCSS, SASS and LESS files by default', function (done) {
+    var config = {
+      context: './test/fixtures/test8',
+      entry: './index'
+    };
+
+    pack(assign({}, baseConfig, config), function (err, stats) {
+      expect(err).to.not.exist;
+      expect(stats.compilation.errors.length).to.equal(4);
+      expect(stats.compilation.warnings.length).to.equal(0);
```
Comment (JaKXz) [inline test/index.js]: Please use the `.to.have.length` assertion on the `errors` and `warnings` arrays, because they provide better messages when failing.

File: test/index.js [hunk 2]
```diff
@@ -120,6 +120,20 @@ describe('sasslint-loader', function () {
     });
   });
 
+  it('should work with CSS, SCSS, SASS and LESS files by default', function (done) {
+    var config = {
+      context: './test/fixtures/test8',
+      entry: './index'
+    };
+
+    pack(assign({}, baseConfig, config), function (err, stats) {
```
Comment (JaKXz) [inline test/index.js]: This method's been updated to return a promise which makes writing these tests a bit easier :) (#27)

```js
return pack(/* config */)
  .then(function (stats) {
    //asserts on stats
  });
```

**Comment by VovanR** [comment]
@JaKXz done!

File: test/fixtures/test9/test.sass [hunk 3]
```diff
@@ -0,0 +1,3 @@
+body {
```
Comment (JaKXz) [inline test/fixtures/test9/test.sass]: Could you try using sass specific syntax here?

File: test/fixtures/test9/test.less [hunk 4]
```diff
@@ -0,0 +1,3 @@
+body {
+  foo
```
Comment (JaKXz) [inline test/fixtures/test9/test.less]: Let's try a LESS-specific feature/syntax here?

**Comment by JaKXz** [comment]
Closing in favour of #107 - thanks for your work on this.

cc @barraponto
